airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -108,6 +108,48 @@ def patch_dag(session, dag_id, update_mask=None):
     session.commit()
     return dag_schema.dump(dag)
 
+@security.requires_access([(permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG)])
+@format_parameters({'limit': check_limit})
+@provide_session
+def patch_dags(limit, session, offset=0, only_active=True, tags=None, dag_id_pattern=None, update_mask=None):
+    """Patch all DAGs."""
+    if only_active:
+        dags_query = session.query(DagModel).filter(~DagModel.is_subdag, DagModel.is_active)
+    else:
+        dags_query = session.query(DagModel).filter(~DagModel.is_subdag)
+
+    if dag_id_pattern:
+        dags_query = dags_query.filter(DagModel.dag_id.ilike(f'%{dag_id_pattern}%'))
+
+    readable_dags = current_app.appbuilder.sm.get_accessible_dag_ids(g.user)
```

Comment:
```suggestion
    editable_dags = current_app.appbuilder.sm.get_editable_dag_ids(g.user)
```

Since `get_accessible_dag_ids` returns any DAGs which are editable _or_ readable.